config/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus
+  labels:
+    app: metrics
+    component: prometheus
+data:
+  prometheus.yml: |-
+    global:
+      scrape_interval: 15s
+    scrape_configs:
+    - job_name: 'prometheus'
+      # Override the global default and scrape targets from this job every 5 seconds.
+      scrape_interval: 5s
+      # metrics_path defaults to '/metrics'
+      # scheme defaults to 'http'.
+      static_configs: 
+      - targets: ['localhost:9090']
+    - job_name: 'sk8s'
+      # Override the global default and scrape targets from this job every 5 seconds.
+      scrape_interval: 5s
+      # metrics_path defaults to '/metrics'
+      # scheme defaults to 'http'.
+      metrics_path: '/prometheus'
+      static_configs:
+      - targets: ['topic-gateway:80','topic-controller:80','event-dispatcher:80','spring-handler:80','greeter:80']
```

Comment:
is there any way to add targets dynamically ("greeter" is just one example of a Function... to an extent same applies for any handlers)

Reply:
I think we cold use `<dns_sd_config>` to query for dns hostnames to include 
- https://prometheus.io/docs/operating/configuration/#dns_sd_config

Reply:
ok, cool - something to keep in mind (maybe create an issue to make the targets dynamic?)